example/android/app/_BUCK

```diff
@@ -35,12 +35,12 @@ android_library(
 
 android_build_config(
     name = "build_config",
-    package = "com.realmreactexample",
+    package = "com.realmtstemplate",
```

Comment:
Is this correct or would the more generic `realmreactexample` make more sense if we are deriving all templates from this one?